submissions/DaniaB24/friends-app/js/app.js

```diff
@@ -0,0 +1,94 @@
+const cardsOfFriends = document.getElementById("allHumans");
+const searchBar = document.getElementById("search");
+const filterMenu = document.getElementById("filter_menu");
```

Comment:
One is named with underscore another one with camelCase. You should follow one style. Usually it is or underscore, or dash - `my-id`